Tasks/AzureFileCopy/task.json

```diff
@@ -281,9 +281,53 @@
   "instanceNameFormat": "$(Destination) File Copy",
   "execution": {
     "AzurePowerShell": {
```

Comment:
Will remove this once we are sure it is not possible for users to move back to legacy flow.